src/components/Input/WriteInput/index.tsx

```diff
@@ -0,0 +1,122 @@
+import Image from 'next/image';
+import type { HTMLAttributes } from 'react';
+import { useMemo, useRef, useState } from 'react';
+import { assignInlineVars } from '@vanilla-extract/dynamic';
+
+import { MAIN_INPUT_MAX_LENGTH } from '@/src/constants/config';
+import type { UseInputReturn } from '@/src/hooks/useInput';
+
+import * as style from './style.css';
+import { inputHeight } from './style.css';
```

Comment:
```suggestion
import * as style from './style.css';
```
위 import로 통일해서 작업해주세요!